src/main/java/se/michaelthelin/spotify/model_objects/special/PlaybackQueue.java

```diff
@@ -0,0 +1,91 @@
+package se.michaelthelin.spotify.model_objects.special;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.gson.JsonObject;
+import se.michaelthelin.spotify.model_objects.AbstractModelObject;
+import se.michaelthelin.spotify.model_objects.IModelObject;
+import se.michaelthelin.spotify.model_objects.miscellaneous.CurrentlyPlaying;
+import se.michaelthelin.spotify.model_objects.specification.*;
+
+import java.util.List;
+
+/**
+ * Includes tracks that are in the queue of the user for the upcoming playback.
```

Comment:
Just use what the api specification lists as well: `Get the list of objects that make up the user's queue.`
Here and below.